controllers/comment.js

```diff
@@ -0,0 +1,88 @@
+const express = require('express');
+const {ValidationError} = require('sequelize');
+
+const router = express.Router();
+const responseUtil = require('../helpers/response');
+const { Comment, User } = require('../models');
+
+const createComment = (req, res) => {
+    try {
+        const {photo_id,comment} = req.body;
+        const {id} = req.user;
+        Comment.create({photo_id,comment,user_id:id})
+            .then((data) => {
+                responseUtil.successResponse(
+                    res,
+                    `Hi your Comment added`,
+                    {photo: {id: data.id, photo_id, comment, user_id: id, updatedAt: data.updatedAt, createdAt: data.createdAt}}
+                );
+            }).catch(err => {
+                if (err instanceof ValidationError)
+                    return responseUtil.validationErrorResponse(res, err.errors[0].message)
+                else
+                    return responseUtil.badRequestResponse(res, err);
+            })
+    } catch (e) {
+        return responseUtil.serverErrorResponse(res, e.message);
+    }
+}
+
+const getComment = (req, res) => {
+    try { 
+       Comment.findAll()
+            .then((data) => {
+                return responseUtil.successResponse(res, null, data)
+            })
+            .catch(err => {
+                return responseUtil.badRequestResponse(res, err);
+            })
+    } catch (e) {
+        return responseUtil.serverErrorResponse(res, e.message);
+    }
+}
+
+const updateComment = (req, res) => {
+    try {
+        const {comment} = req.body;
+        const bodyData = {comment};
+        const id = parseInt(req.params.commentId);
+        Photo.update(bodyData, {where: {id}})
```

Comment:
ini salah panggil, panggilnya harusnya `Comment`, coba diubah yaa
```suggestion
        Comment.update(bodyData, {where: {id}})
```